website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -119,6 +119,8 @@ The following arguments are supported:
 * `os_type` - (Optional) The Operating System used for the Agents. Possible values are `Linux` and `Windows`.  Changing this forces a new resource to be created. Defaults to `Linux`.
 * `vnet_subnet_id` - (Optional) The ID of the Subnet where the Agents in the Pool should be provisioned. Changing this forces a new resource to be created.
 
+~> **NOTE:** There's a known issue where Agents connected to an Internal Network (e.g. on a Subnet) have their network routing configured incorrectly; such that Pods cannot be located across nodes. This is a bug in the Azure API - and will be fixed there in the future.
```

Comment:
There's a known issue where Agents connected to a Network (e.g. on a Subnet) not provisioned as part of the creation of the AKS cluster have their network routing configured incorrectly; such that Pods cannot communicate across nodes.

Reply:
so does the issue only manifest when attaching an existing cluster to an internal subnet? at this time the resource requires a recreation for this kind of change

Reply:
Since the `vnet_subnet_id` is ForceNew, this can only affect New clusters from within Terraform; so I think we're covered on that front. I'll update to include the comment about communication 👍